### DISCUSSION TITLE
電腦的儲存空間被用完要怎麼辦？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#306] [Q&A] [answered]
我這一臺電腦的東西都儲存在D槽，現在因為空間不夠，什麼都存不進去。情況如下。
![未命名](https://user-images.githubusercontent.com/86505454/148646550-0a0e6532-2cb5-4a0e-9e99-94bd496c51fd.png)
不知是否有人碰過這種情況，若有人碰過且知道解決方法，想請你與我分享。謝謝。

**Comment by EricAgumonJou** [comment]
這個影片供你參考～～windows電腦有硬碟清理的功能
https://www.youtube.com/watch?v=2bLp19cBVK8

**Comment by EricAgumonJou** [comment, marked as answer]
如果清理完後使用空間還是相當不足的話，還是建議自己買個外接硬碟，把平時幾乎不會用到的檔案丟到外接硬碟內。
（現在市面上的外接硬碟都不會太貴）供你參考參考～

**Reply by enoch-lightning** [reply]
這應該是我要使用的方式，謝謝你。
